tests/TestCase/Cache/Engine/RedisEngineTest.php

```diff
@@ -33,6 +33,7 @@ public function setUp()
     {
         parent::setUp();
         $this->skipIf(!class_exists('Redis'), 'Redis extension is not installed or configured properly.');
+        $this->skipIf(version_compare(PHP_VERSION, '7.2.0beta', '>='), 'Redis is misbehaving in PHP7.2');
```

Comment:
@markstory the version used is 7.2.0-dev see https://travis-ci.org/cakephp/cakephp/jobs/262358142#L505